### ISSUE TITLE
finalize config strategy for cloud

### ISSUE THREAD

**Issue by jrhizor** [airbytehq/airbyte#6957]
Are we getting rid of nginx for cloud? Are we stabilizing the config file provider? We need to make these decisions and set up infra work / frontend work that needs to be done.

**Comment by davinchia** [comment]
@cgardens and I spoke today. We want to do so and are waiting on better instructions.

**Comment by timroes** [comment]
Closing in favor of https://github.com/airbytehq/airbyte/issues/10955
